coverage/.last_run.json

```diff
@@ -0,0 +1,5 @@
+{
```

Comment:
Try to add each library or new set of files in a separate commit.  This makes it easier to roll back if you need to, and easier for others to read the commit history.